week-2/G-conditionals-2/exercise-3.js

```diff
@@ -9,7 +9,18 @@
 */
 
 function calculateGrade(mark) {
-
+  if (mark >= 80 && mark <= 100) {
```

Comment:
No need for ` mark <= 100` when you know that's the highest grade :) But it's not wrong!